tests/mph_comp_test.py

```diff
@@ -127,4 +135,4 @@ def test_visualize_mph():
     with tempfile.TemporaryDirectory() as temp_dir:
         # test for saving to directory
         mph.visualize_mph(mph_data, out_dir=temp_dir, regression=True)
-        assert os.path.exists(os.path.join(temp_dir, 'fov_vs_mph.png'))
+        assert os.path.exists(os.path.join(temp_dir, "fov_vs_mph.png"))
```

Comment:
bet this ' -> " thing was annoying

Reply:
Thankfully, the pre-commit step will auto-format everything, so I didn't have to do it.